packages/baguette_sharp/baguette_sharp.2.2.1-1/opam

```diff
@@ -0,0 +1,49 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "The Baguette# Interpreter REPL"
+description: "The REPL for Baguette#"
+maintainer: ["Charlotte Thomas"]
+authors: ["Charlotte Thomas"]
+license: "GPL-3.0-or-later"
+homepage: "https://baguettesharp.fr"
+doc: "https://github.com/coco33920/ocaml-baguettesharp-interpreter/wiki"
+bug-reports:
+  "https://github.com/coco33920/ocaml-baguettesharp-interpreter/issues"
+depends: [
+  "dune" {>= "3.0"}
+  "ocaml" {>="4.13.1"}
+  "fmt" {>="0.7.0"}
+  "linenoise" {>="1.4.0"}
+  "odoc" {with-doc}
+]
+patches: ["dune-project-syntax.patch"]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo:
+  "git+https://github.com/coco33920/ocaml-baguettesharp-interpreter.git"
+url {
+  src:
+    "https://github.com/coco33920/ocaml-baguettesharp-interpreter/archive/refs/tags/2.2.1.tar.gz"
+  checksum: [
+    "md5=7a7d67de03c9d8d89c4b73a06bd5ae49"
+    "sha512=55eac5ce65f7c7fede1abda1c9bd9132f6006b8d5ec909116805ca75076c84bcf06635af06956c82e3b53191a62417c6aadca9fa27ce69ef72d42987d6afbc52"
+  ]
+}
+extra-source "dune-project-syntax.patch" {
+  src: "https://patch-diff.githubusercontent.com/raw/vanilla-extracts/ocaml-baguettesharp-interpreter/pull/34.patch"
```

Comment:
I suggest to move this to [`opam-source-archives`](https://github.com/ocaml/opam-source-archives/tree/main/patches) if that is ok with you